uqcsbot/scripts/events.py

```diff
@@ -6,42 +6,55 @@
 from pytz import timezone, utc
 from typing import Tuple, Optional
 from uqcsbot import bot, Command
-from uqcsbot.utils.command_utils import UsageSyntaxException
+from uqcsbot.utils.command_utils import UsageSyntaxException, loading_status
 from uqcsbot.utils.itee_seminar_utils import (get_seminars, HttpException, InvalidFormatException)
 
-CALENDAR_URL = "https://calendar.google.com/calendar/ical/q3n3pce86072n9knt3pt65fhio%40group.calendar.google.com/public/basic.ics"  # noqa
-FILTER_REGEX = re.compile('(full|all|[0-9]+( weeks?)?)')
+CALENDAR_URL = ("https://calendar.google.com/calendar/ical/"
+                "q3n3pce86072n9knt3pt65fhio%40group.calendar.google.com/public/basic.ics")
+FILTER_REGEX = re.compile('full|all|[0-9]+( weeks?)?|jan.*|feb.*|mar.*|apr.*|may.*|jun.*'
+                          '|jul.*|aug.*|sep.*|oct.*|nov.*|dec.*')
 BRISBANE_TZ = timezone('Australia/Brisbane')
-MONTHS = ["JAN", "FEB", "MAR", "APR", "MAY", "JUN", "JUL", "AUG", "SEP", "OCT", "NOV", "DEC"]
+# empty string to one-index
+MONTH_ABBR = ["", "JAN", "FEB", "MAR", "APR", "MAY", "JUN",
+              "JUL", "AUG", "SEP", "OCT", "NOV", "DEC"]
+MONTH_FULL = ["", "January", "February", "March", "April", "May", "June",
+              "July", "August", "September", "October", "November", "December"]
+MONTH_NUMBER = {"jan": 1, "feb": 2, "mar": 3, "apr": 4, "may": 5, "jun": 6,
```

Comment:
Can get from index of previous arrays, at worst, build it programatically:
`{month.lower():index for index, month in enumerate(MONTH_ABBR)}`

Reply:
Fixed.